src/libraries/System.Net.Security/src/System/Net/Security/SslStream.Implementation.cs

```diff
@@ -387,6 +387,49 @@ private void ForceAuthentication(bool receiveFirst, byte[] buffer, AsyncProtocol
             }
         }
 
+        internal async Task ForceAuthenticationAsync(bool receiveFirst, byte[] buffer, CancellationToken cancellationToken)
+        {
+            _Framing = Framing.Unknown;
```

Comment:
As long as you're making changes, can you rename _Framing => _framing?

Reply:
updated.